<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="chooseFileButton">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vyberte soubor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filePath">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cesta souboru k podepsání</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>300</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Podepiš</string>
    </property>
   </widget>
   <widget class="QPushButton" name="generateKeyFilesButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>240</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vygenerovat soubory s klíči</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseKeyFileButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Vyberte soubor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyPath">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label1_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>120</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Klíč pro zašifrování / dešifrování RSA</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="signRadio">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Podepsání</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QRadioButton" name="verificationRadio">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Ověření</string>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QPushButton" name="verifyButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>340</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Ověřit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>561</width>
      <height>44</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
     </font>
    </property>
    <property name="text">
     <string>DSA - elektronický podpis</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>280</y>
      <width>191</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Název a přípona souboru:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Datum poslední změny souboru:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Velikost souboru v bytech:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Name_suffix">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Date">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>310</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="File_size">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>340</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
